extensions/wikibase/module/langs/translation-en.json

```diff
@@ -34,9 +34,8 @@
     "wikibase-addition/version-error": "Manifest version $1 is not supported. OpenRefine supports manifest versions 1.x and 2.x only.",
     "wikibase-schema/dialog-header": "Align to Wikibase",
     "wikibase-schema/target-wikibase-instance": "Target Wikibase instance:",
-    "wikibase-schema/dialog-explanation": "The schema below specifies how your tabular data will be transformed into <a href=\"$1\">$2</a> edits.",
     "wikibase-schema/start-from-an-existing-schema": "Start from an existing schema:",
-    "wikibase-schema/save-new-schema": "Save new...",
+    "wikibase-schema/save-new-schema": "Save schema edits",
```

Comment:
"schema edits" seems confusing in this context: that could also refer to "the Wikibase edits generated by the schema". What about "schema changes"?

Reply:
Sound good to me, @lozanaross?